test/pytest/mockwebserver.py

```diff
@@ -0,0 +1,492 @@
+# This file is part of Cockpit.
```

Comment:
The intent is to run this like `spy -m test.pytest.mockwebserver` which will hit the `__name__ == '__main__'` case and doesn't require the `#!`.  If you run it directly then it won't be able to find its relative imports, much less the bits of cockpit it imports.

This could use docs :)